app/scripts/main.js

```diff
@@ -519,6 +531,7 @@ document.addEventListener("DOMContentLoaded", function () {
                 }).then(function(){
                     resolve(options);
                 }).catch(function (error) {
+                    console.log(`configImporter error (Global scope). Message: ${error.message}`);
```

Comment:
Remove this

Reply:
This is how the multicall problem in configImporter was debugged. will remove. done